mad-dog/README.md

```diff
@@ -1,24 +1,20 @@
 # mad-dog
-🐶 Creator Node Test Suite
+🐶 End-to-End Protocol Test Suite (mostly covers Content Node)
```

Comment:
let's actually remove the `(mostly covers Content Node)` since this isn't the goal